src/main/kotlin/lemmy/client/Library.kt

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
I'm not sure if we should leave these, or just rely on our own tests?

Reply:
can probably remove. the tests that are there now should be fine. i need to figure out github actions, never used it before ^_^